data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -195,13 +165,16 @@
             <column name="file_path" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="content" type="TEXT">
+            <column name="cpi_name" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="changeset_id" type="UUID">
+            <column name="cpi_version" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="entity_version" type="INT">
+            <column name="cpi_signer_summary_hash" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Do we need these? And do we not need the changeset id (instead)?

Reply:
I removed the cpi related columns, and also the changeset id, audit table becomes more of an "append-only" history of changelogs and CPK file checksums.